projects/media-viewer/src/lib/annotations/annotations.component.spec.ts

```diff
@@ -70,20 +70,30 @@ describe('AnnotationsComponent', () => {
   it('deletes a comment', async () => {
     const commentElements = debugElement.nativeElement.querySelectorAll('form.aui-comment');
 
-    commentElements[0].click();
+    commentElements[1].click();
     fixture.detectChanges();
 
-    return new Promise(resolve => {
+    await new Promise(resolve => {
       component.update.subscribe(as => {
-        expect(as.annotations[0].comments.length).toBe(0);
+        expect(as.annotations[1].comments.length).toBe(0);
 
         resolve();
       });
 
-      const buttons = commentElements[0].querySelectorAll('button');
+      const buttons = commentElements[1].querySelectorAll('button');
       buttons[1].click();
     });
-
   });
 
+  it('updates a comment', async () => {
+    await new Promise(resolve => {
+      component.update.subscribe(as => {
+        expect(as.annotations[0].comments[0].content).toBe('Updated comment');
+
+        resolve();
+      });
+
+      component.updateComment(0, 'Updated comment');
```

Comment:
is the previous section setting up a promise, which doesn't get resolved until this runs? What happens if this line isn't run, does the test fail, or just stay open indefinitely? Is there a timeout or something?